src/schemas/ClusterRoutingTableResponse.yaml

```diff
@@ -0,0 +1,16 @@
+type: object
+properties:
+  results:
+    type: array
+    items:
+      type: object
+      properties:
+        destination:
+          type: string
+          nullable: false
```

Comment:
not nullable

Reply:
I set the nullable parameter as false, do you want me to remove the parameter ? or are you talking about the description parameter?